lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -208,34 +169,37 @@ define(function(require) {
 
 
     /**
-     * Check to see if there are any reportback items currently in the
-     * inventory after each click of the View More button.
-     * If there are items already, then append them, otherwise if there are
-     * more in storage then request them to add to the inventory.
-     * If there's low inventory and no more in storage then remove the button.
+     * Add the View More button if there are additional reportbacks.
      */
-    evaluateInventory: function () {
-      if (this.inventory.length) {
-        if (this.lowInventory()) {
-          this.request(this.currentIndex);
+    enableViewMore: function () {
+      var _this = this;
+      var buttonText = Drupal.t('View more');
 
-          if ((this.currentIndex += this.bundleSize) >= this.itemsTotal) {
-            this.currentIndex = this.itemsTotal;
-          }
-        }
-        else {
-          this.appendEntries(this.currentIndex);
-          this.currentIndex += this.bundleSize;
-        }
+      this.$viewMoreContainer = $('<div class=\'reportback__view-more\'></div>');
+      this.$viewMoreButton = $('<button id=\'view-more-button\' class=\'button button--view-more -tertiary inline-alt-text-color\'>' + buttonText + '</button>');
```

Comment:
Another good case for a template string, since you can avoid having to escape the `'`s!